### PR TITLE
Replace generic motors with CAN motors

### PR DESCRIPTION
**Pull request by KenwoodFox**
And other cleanups

### CODE REVIEW

File: rio/subsystems/drivesubsystem.py [hunk 1]
```diff
@@ -66,7 +84,7 @@ def __init__(self) -> None:
         self.leftEncoder.setDistancePerPulse(encoderDistPerP)
         self.rightEncoder.setDistancePerPulse(encoderDistPerP)
 
-        self.gyro = wpilib.ADXRS450_Gyro()
+        self.gyro = wpilib.ADXRS450_Gyro()  # We may need to use another gyro
```
Comment (KenwoodFox) [inline rio/subsystems/drivesubsystem.py]: This gyro comes from the boilerplate code, consider using the [pigeon from last year](https://github.com/FRC-1721/1721-RapidReact/blob/a8ba43fd3ed713b8fc7e0688153e1a2970b03d00/rio/subsystems/drivetrain.py#L58-L68)

**Comment by KenwoodFox** [comment]
Did we get a running driving robot?

**Comment by Kredcool** [comment]
No, it should just need to be deployed. We didn't get it running because we added the other two motors and there were a few code issues.

**Comment by KenwoodFox** [comment]
@Kredcool If this fails again today, please [make an issue](https://github.com/FRC-1721/1721-ChargedUp/issues/new) and include all the details and pictures you can. Issues are a great way to track..

well.. issues lol.

Good luck!

**Review by Kredcool — APPROVED** [review]

**Pull request merged**